xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -1278,10 +1294,10 @@
         <Parameter Name="value" Type="System.Int64" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">The span of bytes where the value is to be written as big endian.</param>
+        <param name="value">The value to write into the span of bytes.</param>
+        <summary>Writes an <see cref="T:System.Int64" /> into a span of bytes as big endian.</summary>
+        <returns><see langword="true" /> if the span can contain an <see cref="T:System.Int64" />; otherwise, <see langword="false" />.</returns>
         <remarks>To be added.</remarks>
```

Comment:
nit: Add remark similar to read, that we write **X bytes** to the **beginning**.

```suggestion
        <remarks>Writes exactly 8 bytes to the beginning of the span.</remarks>
```